frameworks/projects/Jewel/src/main/royale/org/apache/royale/jewel/beads/controls/combobox/ComboBoxItemByField.as

```diff
@@ -0,0 +1,71 @@
+package com.iest.winplusweb.beads
+{
+  import org.apache.royale.core.IBead;
+  import org.apache.royale.jewel.ComboBox;
+  import org.apache.royale.core.IStrand;
+  import org.apache.royale.collections.CollectionUtils;
+  import org.apache.royale.collections.ArrayList;
+  import org.apache.royale.events.Event;
+
+  public class ComboBoxItemByField implements IBead{
+
+      protected var comboBox:ComboBox;
+
+
+      public function ComboBoxItemByField()
+		{
+		}
+
+        public function set strand(value:IStrand):void
+		{
+			comboBox = value as ComboBox;
```

Comment:
Have you considered casting this to an interface so it can work on strands other than jewel ComboBox? It could be better to place this in Basic so other component sets can use this.

Reply:
Ok, it's a great idea!!!

Reply:
Hi:

I have tried to change the code to use an interface but I have not got it to work. 

I will send an email to the list with the code to see if you can help me.

Thanks!